build.sbt

```diff
@@ -3,10 +3,12 @@ import Dependencies._
 
 name := "play-googleauth"
 
+ThisBuild / scalaVersion := "2.13.11"
```

Comment:
Why not 2.13.12?

Reply:
Vague justification at https://github.com/guardian/play-secret-rotation/pull/411#discussion_r1389064370 !